imperatives/imperatives.go

```diff
@@ -184,8 +186,8 @@ func Apply(table Table, cmd string) error {
 
 func readAddAgg(s *toki.Scanner, table Table) error {
 	t := s.Next()
-	if t.Token != sumFn && t.Token != avgFn && t.Token != minFn && t.Token != maxFn && t.Token != lastFn && t.Token != deltaFn && t.Token != stdevFn {
-		return errors.New("invalid function. need avg/max/min/sum/last/delta/stdev")
+	if t.Token != sumFn && t.Token != avgFn && t.Token != minFn && t.Token != maxFn && t.Token != lastFn && t.Token != deltaFn && t.Token != deriveFn && t.Token != stdevFn {
```

Comment:
Is there a better way to doing this?
Something like: 
```
if t.Token not in [sumFn, avgFn, minFn, maxFn, lastFn, deltaFn, deriveFn, stdevFn] {
...
}
```

Reply:
unfortunately go doesn't have something like that, and i saw your approach which i wasn't entirely sold by. we could split it up into a case statement but then it would take many lines.

Reply:
That's unfortunate.